scripts-local/tests/gaia_tests.bats

```diff
@@ -5,7 +5,7 @@ setup_file() {
   SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
   PATH="$SCRIPT_DIR/../../:$PATH"
 
-  #set allows us to export all variables in account_vars
+  # set allows us to export all variables in account_vars
```

Comment:
any idea why this line keeps getting uncommented?

Reply:
not a clue, but I'll keep an eye on it - this is the first time I noticed it